rca/account_management/views.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib.auth.models import Group
+from django.views.generic import FormView
+
+from rca.users.models import User
+
+from .forms import StudentCreateForm
+
+
+class CreateStudentFormView(FormView):
+
+    template_name = "account_management/create.html"
+    form_class = StudentCreateForm
+    success_url = "/admin/student/create"  # TODO - better success template
```

Comment:
Probably back to the create view with a message added to the admin

Reply:
Sounds good, especially if they might be creating users in a batch